sw/airborne/modules/loggers/sdlog_chibios/usbStorage.c

```diff
@@ -205,16 +191,15 @@ static const USBDescriptor* getDescriptor(USBDriver* usbp, uint8_t type, uint8_t
 }
 
 /* Handles global events of the USB driver */
-static void usbEvent(USBDriver* usbp, usbevent_t event)
+static void usbEvent(USBDriver *usbp, usbevent_t event)
 {
   (void) usbp;
 
-  switch (event)
-  {
+  switch (event) {
     case USB_EVENT_CONFIGURED:
-      chSysLockFromIsr();
+      chSysLockFromISR();
```

Comment:
Since we are using ChibiOS functions here we should have checks that prevent compiling this module without ChibiOS.
Also, if I understand it correctly, we want to have the core code universal (i.e. no RTOS specific code), but the modules can be defined for specific architectures only? Is that correct @flixr ?